Numpy 2.4 raises error when setting a single element sequence to an array element.

From numpy 2.4 (https://github.com/numpy/numpy/releases/tag/v2.4.0)

> ## Raise TypeError on attempt to convert array with ndim > 0 to scalar
> Conversion of an array with ndim > 0 to a scalar was deprecated in NumPy
>1.25. Now, attempting to do so raises TypeError. Ensure you extract a
> single element from your array before performing this operation.


This effects most of the imas tests see https://github.com/Fusion-Power-Plant-Framework/eqdsk/actions/runs/20442319259/job/58737948826:
<details><summary>Test failures</summary>
<p>

```bash
FAILED tests/test_imas.py::test_imas_write_read[imas-None-True-jetto.eqdsk_out-jetto] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[imas-None-True-DN-DEMO_eqref_withCoilNames.json-3] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[imas-3.42.0-False-jetto.eqdsk_out-jetto] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[imas-3.42.0-False-DN-DEMO_eqref_withCoilNames.json-3] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[imas-4.0.0-False-jetto.eqdsk_out-jetto] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[imas-4.0.0-False-DN-DEMO_eqref_withCoilNames.json-3] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[imas-4.1.0-True-jetto.eqdsk_out-jetto] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[imas-4.1.0-True-DN-DEMO_eqref_withCoilNames.json-3] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[None-None-True-jetto.eqdsk_out-jetto] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[None-None-True-DN-DEMO_eqref_withCoilNames.json-3] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[None-3.42.0-False-jetto.eqdsk_out-jetto] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[None-3.42.0-False-DN-DEMO_eqref_withCoilNames.json-3] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[None-4.0.0-False-jetto.eqdsk_out-jetto] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[None-4.0.0-False-DN-DEMO_eqref_withCoilNames.json-3] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[None-4.1.0-True-jetto.eqdsk_out-jetto] - ValueError: setting an array element with a sequence.
FAILED tests/test_imas.py::test_imas_write_read[None-4.1.0-True-DN-DEMO_eqref_withCoilNames.json-3] - ValueError: setting an array element with a sequence.
```

</p>
</details>

--- a/eqdsk/imas.py
+++ b/eqdsk/imas.py
@@ -245,7 +245,7 @@ def to_imas(  # noqa: PLR0915
     equilibrium_ids.ids_properties.homogeneous_time = 1
     global_quantities.ip = eqdsk.cplasma
     vacuum_toroidal_field.b0.resize(time_index + 1)
-    vacuum_toroidal_field.b0[time_index] = np.array([eqdsk.bcentre] * (time_index + 1))
+    vacuum_toroidal_field.b0 = np.array([eqdsk.bcentre] * (time_index + 1))
     vacuum_toroidal_field.r0 = eqdsk.xcentre
     global_quantities.psi_boundary = eqdsk.psibdry
     global_quantities.psi_magnetic_axis = eqdsk.psimag

